src/components/AssetSearch/AssetList.tsx

```diff
@@ -1,13 +1,18 @@
 import { ListProps } from '@chakra-ui/react'
 import { Asset } from '@shapeshiftoss/types'
+import * as _ from 'lodash'
```

Comment:
Don't import all exports from `lodash` package, this won't tree-shake, see https://lodash.com/per-method-packages
```suggestion
import sortBy from 'lodash/sortBy'
```